### PR TITLE
Re{factor/structure} the UI package

### PR DESCRIPTION
**Pull request by Fnux**
Refactor the `ui` package for easier and sexier sharing between the event 'showcase' and 'administration'.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,6 +4,7 @@ buildscript {
     repositories {
         google()
         jcenter()
+
```
Comment (betty) [inline build.gradle:7]: ?

**Comment by betty** [comment]
This is an enormous PR. It's hard to review 1000+ loc (even if a lot of the files are changed just for the sake of renaming an import). Try to have smaller commits.

**Comment by Fnux** [comment]
This PR only moves things arounds and improve the `EventAdministrationActivity`. I merged master into this branch since (rebase was going to be tedious) but github got confused and displays quite a lot of changes.

**Comment by Fnux** [comment]
Hum, nevermind. This branch was opened against `112-user-permissions` which got merged into another branch then into master. I'll have to reopen it against master. I'm used to github being half-dumb compared to gitlab.

**Pull request closed without merging**